install_gpg_component.sh

```diff
@@ -285,6 +285,8 @@ if [[ "${_arg_version}" =~ ^latest ]]; then
 	determine_latest_version
 fi
 
+[[ "${_arg_sudo}" = "on" ]] && sudo=sudo || sudo=
```

Comment:
@dewyatt How about moving this condition to `build_and_install()` in order to avoid another global variable?

Reply:
Good idea, done!